sdk/monitor/query/MIGRATION.md

```diff
@@ -0,0 +1,211 @@
+# Guide to migrate from `azquery` to `azlogs`, `azmetrics`, and `armmonitor`
+
+With the service team's creation of a new data plane metrics endpoint, the Go SDK team decided to reimagine our Monitor story. After careful deliberation, we decided to restructure Monitor Query to more accurately reflect the [REST API][rest_api].
+
+Logs largely remained the same; the code just moved from `azquery` to [`azlogs`][azlogs].
+
+Metrics has a larger split. Users wanting to use the existing ARM API's should use the [`armmonitor`][armmonitor] module. `armmonitor` contains all the metrics functionality from `azquery` and more.
+
+[`azmetrics`][azmetrics] contains the [new data plane metrics][azmetrics_blog] functionality. This new code was not previously generally available in `azquery`. The code allows users to query data from one or multiple resource IDs using a data plane endpoint. `azmetrics` will expand in the future when the service adds more features.
+
+### Name changes
+
+| old module   | old method name |new module | new method name | 
+| ----------- | ----------- | --- | --- |
+| `azquery` | LogsClient.QueryWorkspace | `azlogs` | Client.QueryWorkspace |
```

Comment:
Code-fence method names in this table

Reply:
fixed with [c79390d](https://github.com/Azure/azure-sdk-for-go/pull/22753/commits/c79390ddba46a599c6717d0bde75b9d3ccb46996)